DolbyIO/Source/Private/DolbyIOSubsystem.cpp

```diff
@@ -208,6 +208,40 @@ void UDolbyIOSubsystem::Initialize(const FString& Token)
 			            BroadcastEvent(OnVideoTrackRemoved, VideoTrack);
 		            });
 	        })
+	    .then(
+	        [this](event_handler_id)
+	        {
+		        return Sdk->device_management().add_event_handler(
+		            [this](const audio_device_changed& Event)
+		            {
+			            auto ChangedDirection = ToEDolbyIODeviceDirection(Event.utilized_direction);
+			            if (!Event.device)
+			            {
+				            DLB_UE_LOG("Audio device changed to NO device for direction: %d", int(ChangedDirection));
+				            BroadcastEvent(OnCurrentAudioDeviceChanged, true, FDolbyIOAudioDevice{}, ChangedDirection);
```

Comment:
This `true` and the `false` below are magic values unless you know the layout of the event arguments. I was initially confused about this. I suggest `constexpr bool bIsNone = true;`.